leafmap/common.py

```
@@ -3583,6 +3587,20 @@ def get_local_tile_url(
+    # Route tile URLs through the jupyter-loopback comm bridge so that rasters
+    # render in webview-based frontends (VS Code, Colab, Solara, marimo, etc.)
+    # and in containerized/remote Jupyter environments where the browser cannot
+    # reach the jupyter-server origin directly. The `get_leaflet_tile_layer` and
+    # `get_folium_tile_layer` helpers used by `get_local_tile_layer` do this
+    # automatically, but a bare `TileClient` does not. See
+    # https://github.com/opengeos/leafmap/issues/1331
+    if hasattr(client, "enable_jupyter_loopback"):
+        try:
+            client.enable_jupyter_loopback()
+        except Exception:
+            pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)